runtime/criusupport/criusupport.cpp

```diff
@@ -638,6 +689,13 @@ Java_org_eclipse_openj9_criu_CRIUSupport_checkpointJVMImpl(JNIEnv *env,
 		/* We can only end up here if the CRIU restore was successful */
 		isAfterCheckpoint = TRUE;
 
+		if (false == loadRestoreArguments(currentThread, optionsFileChars)) {
```

Comment:
Need to check `NULL != optionsFile` to avoid `loadRestoreArguments` w/ an empty buffer.

Reply:
`addXOptionsFile` handles the empty `optionsFileChars`. I want to call `loadRestoreArguments` uncondiontally because Ill have a follow on PR that also looks for options in the env variables (IBM_JAVA_OPTIONS, etc.)

Reply:
`OPENJ9_JAVA_OPTIONS` right?  =)